### PR TITLE
More general check for graveyard 

### PR DESCRIPTION
**Pull request by piperlincoln**
Updated GraveIdentifyAndRemove.py to include a more general check for a graveyard EntitySet. Instead of searching for a Name Tag value of "graveyard", it checks if that substring exists within the Name Tag value as a whole. This is helpful in cases such as the UWUW workflow, when the Name Tag value is `mat:Graveyard`. (Specified in #11)

### CODE REVIEW

**Review by pshriwise — APPROVED** [review]
This looks good @piperlincoln! Thanks!! I'll wait until the end of the day to merge so others can have a look if they want.

**Review by kkiesling — APPROVED** [review]
LGTM as well

**Pull request merged**
